jobs/adspert_python_dev.html

```diff
@@ -7,42 +7,34 @@
 contact:
     name: Karl Bartel
     email: jobs@adspert.net
-created: !!timestamp '2016-05-08'
+created: !!timestamp '2017-05-15'
 tags:
   - python
   - sql
 ---
+We are looking for a **Python Software Developer (m/f)**
 
 **Note: The position is full-time and based in our Berlin office, therefore if you are outside of Berlin you must be prepared to relocate for this role.**
 
-Adspert was founded in Berlin in 2010 and our innovative technology launched just one year later. Based on trading systems, Adspert automatically organizes and optimizes online advertising. Bertelsmann Digital Media Investment and the Bell Group's support Adspert as strategic partners and investors.
+Adspert was founded in Berlin in 2010 and our innovative technology was launched just one year later. Based on trading systems, Adspert automatically organizes and optimizes online advertising. Bertelsmann Digital Media Investment and K-Invest support Adspert as strategic partners and investors.
 
-Today, around 20 Adspert employees are hard at work in the heart of Berlin, passionately implementing the vision of developing bid management of the future.
+**Our team** in the centre of Berlin is passionately working on bid management systems of the future. With us you can expect a friendly atmosphere and a healthy work-life balance.
 
-**We offer:**
+**We develop** our solutions with Python, Go, JavaScript and PostgreSQL and mainly work on Linux and Mac. Distributed version control, good test coverage and automated deployment are fundamental to our development process.
 
-We develop our solutions with Python, Go, JavaScript and PostgreSQL and mainly work on Linux and Mac. Distributed version control, test-driven development and automated deployment processes are basic working methods. With us, you can expect a friendly atmosphere and a healthy attitude to the balance between work and leisure.
+**You are** a motivated developer with a couple of years of professional experience who is excited about analyzing problems and the development of appropriate solutions.
```

Comment:
"a couple" literally means 2. I've noticed that some non-native English speakers often use the word to mean "a few". Perhaps what you meant to say here is "at least two"